lib/hammer_cli_foreman.rb

```diff
@@ -3,6 +3,15 @@
 require 'apipie_bindings'
 
 module HammerCLIForeman
+  @compute_resources = {}
```

Comment:
why are you moving in from lib/hammer_cli_foreman/compute_resource/register_compute_resources.rb ?
there are plugins that depend on these methods, and this is not so good to change those